apps/vercel/frontend/src/components/config-screen/ContentTypePreviewPathSection/InformationalModal/InformationalTables/InfoTable/InfoTable.styles.ts

```diff
@@ -0,0 +1,38 @@
+import tokens from '@contentful/f36-tokens';
+import { css } from 'emotion';
+import { INFORMATIONAL_MODAL_COLUMN_WIDTH } from '../../../../../../constants/styles';
```

Comment:
Don't we / shouldn't we use the thing where we have named top level packages to avoid these long directory traversals?

Reply:
yeah great callout, I have added the aliases to our `tsconfig` here, and will continue to refine other files with this as I continue frontend vercel work 👍